keps/sig-auth/3221-structured-authorization-configuration/README.md

```diff
@@ -499,10 +499,15 @@ Labels {along with possible values}:
 - `name`
 - `code` {"incomplete_request", "bad_response"}
 
-5. `apiserver_authorization_configuration_reload_last_timestamp_seconds`
+5. `apiserver_authorization_config_controller_automatic_reload_last_timestamp_seconds`
 
-This Gauge metric will record last time in seconds when an authorization reload was performed.
-- `status` {success, fail}
+This Gauge metric will record last time in seconds when an authorization reload was performed, partitioned by apiserver_id_hash.
+- `apiserver_id_hash`
+
+6. `apiserver_authorization_config_controller_automatic_reload_failures_total` and `apiserver_authorization_config_controller_automatic_reload_success_total`
+
```

Comment:
nit: `apiserver_authorization_config_controller_automatic_reloads_total` with `type` dimension with an enum value of `success` and `failure`, is slightly easier to code up ratios with.

Reply:
I like this convention. Could we get this documented (https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/instrumentation.md)? I've had some questions about this and it would be ideal if we could point to something definitive.

Reply:
For posterity, here's another place this got discussed: https://github.com/kubernetes/enhancements/pull/4416#discussion_r1473080773